docs/install-upgrade.md

```diff
@@ -36,12 +35,81 @@ You can deploy the MongoDB Community Kubernetes Operator with different scopes b
 
 You scope the Operator to a namespace. The Operator watches MongoDBCommunity resources in that same [namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/).
 
-This is the default scope when you install the Operator using the [installation instructions](#procedure).
+This is the default scope when you [install the Operator using Helm](#install-the-operator-using-helm) or [install the Operator using kubectl](#install-the-operator-using-kubectl).
 
 #### Operator in Different Namespace Than Resources
 
 You scope the Operator to a namespace. The Operator watches MongoDBCommunity resources in other namespaces.
 
+To deploy the Operator in a different namespace than the resources, [Install in a Different Namespace using Helm](#install-in-a-different-namespace-using-helm) or [Install in a Different Namespace using kubectl](#install-in-a-different-namespace-using-kubectl).
+
+### Install the Operator using Helm
+
+You can install the Operator using the [MongoDB Helm Charts](https://mongodb.github.io/helm-charts/).
+
+#### Prerequisites to Install using Helm
+
+Before you install the MongoDB Community Kubernetes Operator using Helm, you must:
+
+1. Have a Kubernetes solution available to use.
+   If you need a Kubernetes solution, see the [Kubernetes documentation on picking the right solution](https://kubernetes.io/docs/setup). For testing, MongoDB recommends [Kind](https://kind.sigs.k8s.io/).
+2. Clone this repository.
+   ```
+   git clone https://github.com/mongodb/mongodb-kubernetes-operator.git
+   ```
+
+#### Procedure using Helm
+
+Use one of the following procedures to install the Operator using Helm:
+
+- [Install in the Default Namespace using Helm](#install-in-the-default-namespace-using-helm)
+- [Install in a Different Namespace using Helm](#install-in-a-different-namespace-using-helm)
+
+##### Install in the Default Namespace using Helm
+
+To install the Custom Resource Definitions and the Community Operator in 
+the `default` namespace using Helm, run the install command from the 
+terminal:
+   ```
+   helm install community-operator mongodb/community-operator
+   ```
+
+If you already installed the `community-operator-crds` Helm chart, you must
+include `--set community-operator-crds.enabled=false` when installing the Operator:
+   ```
+   helm install community-operator mongodb/community-operator --set community-operator-crds.enabled=false
+   ```
+
+##### Install in a Different Namespace using Helm
+
+To install the Custom Resource Definitions and the Community Operator in 
+a different namespace using Helm, run the install 
+command with the `--namespace` flag from the terminal. Include the `--create-namespace` 
+flag if you are creating a new namespace.
+   ```
+   helm install community-operator mongodb/community-operator --namespace mongodb [--create-namespace]
+   ```
+
+### Install the Operator using kubectl
+
+You can install the Operator using kubectl instead of Helm.
```

Comment:
this references a specific command, so I think it should be monospaced
```suggestion
You can install the Operator using `kubectl` instead of Helm.
```